src/Display.tsx

```diff
@@ -144,7 +134,14 @@ const VictimDetail: React.FC<VictimDetailProps> = props => {
     .then(response => response.json())
     .then(jsonData => {
       if (jsonData.features.length) {
+        let neighbouringLocations = [];
+        for (let i = 1; i < 2; i++) {
+          neighbouringLocations.push(jsonData.features[i].place_name);
+        }
```

Comment:
This for loop runs for only one iteration.

Reply:
It is because, We want only one neighbouring location.

If all the locations are required. Then we can use all the locations in the list.

Loop is just to print limited locations.

Reply:
Okay, I think it is better to keep it simple for now and display only one location. Let's go with using index directly.
We will switch to using for loop if we decide to display multiple locations later.